pickhardtpayments/OracleChannel.py

```diff
@@ -17,18 +17,30 @@ def __init__(self, channel: Channel, actual_liquidity: int = None):
             self._actual_liquidity = random.randint(0, self.capacity)
 
     def __str__(self):
-        return super().__str__()+" actual Liquidity: {}".format(self.actual_liquidity)
+        return super().__str__() + " actual Liquidity: {}".format(self.actual_liquidity)
 
     @property
     def actual_liquidity(self):
         """
         Tells us the actual liquidity according to the oracle. 
 
-        This is usful for experiments but must of course not be used in routing and is also
-        not a vailable if mainnet remote channels are being used.
+        This is useful for experiments but must of course not be used in routing and is also
+        not available if mainnet remote channels are being used.
         """
         return self._actual_liquidity
 
+    @actual_liquidity.setter
+    def actual_liquidity(self, amt: int):
+        """Sets the liquidity of a channel in the Oracle Network
+
+        :param amt: amount to be assigned to channel liquidity
+        :type amt: int
+        """
+        if 0 <= amt <= self.capacity:
+            self._actual_liquidity = amt
+        else:
+            raise ValueError("Oops! The amount to be assigned to channel liquidity is negative or higher than capacity")
```

Comment:
I think we can make the error message more helpful by also adding the amount and the information about the channel to the error.  This would help a user to understand where in their data something broke